package.json

```diff
@@ -64,11 +64,13 @@
     "@types/jest": "^28.1.2",
     "@types/react": "~17.0.21",
     "@types/react-native": "0.70.0",
+    "@types/express": "^4.17.13",
     "commitlint": "^17.0.2",
     "del-cli": "^5.0.0",
     "eslint": "^8.4.1",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
+    "express": "^4.18.0",
```

Comment:
Decided to run an express server instead for the tests because I couldn't get `xhr-mock` to easily work